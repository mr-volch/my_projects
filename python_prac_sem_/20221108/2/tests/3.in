[1,2,3,4,5,1,3]
kucher
(1,1),(2,2),(-1000,-1000)
(100,0),(100,1),(200,200)

