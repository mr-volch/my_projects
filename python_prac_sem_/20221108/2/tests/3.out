Invalid input
Invalid input
Not a triangle
50.00
